modules/era/sql/era_mob_droplist.sql

```diff
@@ -22495,7 +22495,7 @@ INSERT INTO `mob_droplist` VALUES (2743,2,0,1000,656,0);         -- Beastcoin (S
 INSERT INTO `mob_droplist` VALUES (2744,0,0,1000,841,@VCOMMON);   -- Yagudo Feather (Very Common, 24%)
 INSERT INTO `mob_droplist` VALUES (2744,0,0,1000,841,@VCOMMON);   -- Yagudo Feather (Very Common, 24%)
 INSERT INTO `mob_droplist` VALUES (2744,0,0,1000,841,@VCOMMON);   -- Yagudo Feather (Very Common, 24%)
-INSERT INTO `mob_droplist` VALUES (2745,0,0,1000,498,@VCOMMON);   -- Yagudo Bead Necklace (Very Common, 24%)
+INSERT INTO `mob_droplist` VALUES (2744,0,0,1000,498,@VCOMMON);   -- Yagudo Bead Necklace (Very Common, 24%)
```

Comment:
won't this make them drop no necklaces and just a 4th feather?

Reply:
> won't this make them drop no necklaces and just a 4th feather?

The first column is the dropid, that corresponds to a specific mob group that will have that pool of drops. The itemid is the fifth column (different than the feather itemid). Yeah the column names are not so clear.

Reply:
duh. ignore me.